alerts/solr/README.md

```diff
@@ -0,0 +1,39 @@
+# Alerts for Solr in the Ops Agent
+
+## High Request Errors
+
+If `solr.request.error.count` is high there should be cause for concern as your server could be having issue executing queries. 
```

Comment:
In the descriptions for the ReadMe and in the JSON, could you share what thresholds are being set in the sample alert policy? 

i.e. how is "high" being defined?

Reply:
Updated